pkg/networkmanager/ip-settings.jsx

```diff
@@ -133,8 +133,8 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
         });
 
         // Prevent dialog from closing because of <form> onsubmit event
-        if (event)
-            event.preventDefault();
+        if (ev)
+            ev.preventDefault();
```

Comment:
`ev` should always be defined here, butif not, then `ev?.preventDefault()` is a bit nicer. Anyway, not a blocker.

Reply:
Blah, yes this code is wrong. Happy to clean that up though as there is no rush to merge this.

Reply:
It flakes like ~20%

Reply:
The only network related flake here is [this](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20697-d27c6e47-20240703-081835-fedora-coreos-networking/log.html#52), and that seems to be something else.

So if that fixes stuff, please go ahead and merge -- as I said, this is good enough. Thanks!